spec/lib/inclusion_intersection_validator_spec.rb

```diff
@@ -24,7 +24,7 @@ class BadSetupValidator
   class ListValidator
     include ActiveModel::Validations
     attr_accessor :list
-    validates_with ActiveModelValidatesIntersectionOf::Validator, attributes: [:list], in: VALID_ARRAY, message: "not valid"
+    validates_with ActiveModelValidatesIntersectionOf::Validator, attributes: [:list], within: VALID_ARRAY, message: "not valid"
   end
 
   subject { List.new }
```

Comment:
I updated `subject { List.new }` to `subject { ListValidator.new }` and the test was broken. I was gonna to debug it.